packages/browser-wallet/src/tests-with-chain/chain.test.ts

```diff
@@ -12,32 +12,18 @@ import {
 import {Store} from '../store';
 import {ChainWatcher, FakeChain} from '../chain';
 import {Player} from '../integration-tests/helpers';
-import {
-  ETH_ASSET_HOLDER_ADDRESS,
-  CHAIN_NETWORK_ID,
-  CHALLENGE_DURATION,
-  TRIVIAL_APP_ADDRESS
-} from '../config';
+import {CHAIN_NETWORK_ID, CHALLENGE_DURATION, TRIVIAL_APP_ADDRESS} from '../config';
 
 jest.setTimeout(10_000);
 
 const chain = new ChainWatcher();
 const store = new Store(chain);
 
-const provider = new providers.JsonRpcProvider(`http://localhost:${process.env.GANACHE_PORT}`);
-
-let ETHAssetHolder: Contract;
+let NitroAdjudicator: Contract;
 
 beforeAll(async () => {
   (window as any).ethereum = {enable: () => ['0xfec44e15328B7d1d8885a8226b0858964358F1D6']};
-  chain.ethereumEnable();
-
-  const signer = await provider.getSigner('0x28bF45680cA598708E5cDACc1414FCAc04a3F1ed');
-  ETHAssetHolder = new Contract(
-    ETH_ASSET_HOLDER_ADDRESS,
-    ContractArtifacts.EthAssetHolderArtifact.abi,
-    signer
-  );
+  await chain.ethereumEnable();
 });
```

Comment:
Why is it that `NitriAdjudicator` doesn't need to be initialized here in the same way that `ETHAssetHolder` was?

Reply:
Well the truth is that it does... only the test in question is currently being skipped so it "doesn't matter". I presume the test is being skipped because it fails. But I will double check; and it might be nice to not make it more broken with this changeset 😛

Reply:
0a31d6b